doc/hosts/appservice.md

```diff
@@ -0,0 +1,72 @@
+# Azure App Service for Linux
+
+In [Azure App Service on Linux][] the user has many options to deploy their application, including 
+[continuous deployment][] - in which the source code is uploaded to App Service that in turn builds
+the application. If no custom deployment script is provided, Oryx will run behind the scenes and perform
+the required steps to build and configure the application. 
+
+Even if the application is built outside of App Service, e.g. through an external CI/CD pipeline,
+Oryx is still invoked to detect how to start an application if no start up command was specified.
+
+Here we describe some details of this process, and how you might configure them and fix issues
+if needed. We focus on the specifics for App Service; for how we support each language/runtime in
+general, please refer to their specific entry in our [docs page](../README.md).
+
+[Azure App Service on Linux]: https://docs.microsoft.com/en-us/azure/app-service/containers/app-service-linux-intro
+[continuous deployment]: https://docs.microsoft.com/en-us/azure/app-service/deploy-continuous-deployment?toc=%2fazure%2fapp-service%2fcontainers%2ftoc.json
+
+## Overview
+
+When an application is pushed to App Service, e.g. through local git or GitHub integration, the
+source code will be available in `/home/site/repository`. Through a hook in the git repository,
+after the code is pushed App Service will call Oryx to build the application if a deployment
+script wasn't provided. After the build step the web app is placed in `/home/site/wwwroot`,
+location from which it will be executed.
+
+App Service has two components that are relevant in this discussion: the [Kudu][] service,
+where the build happens, and the runtime environments where the web apps are executed. Both environments
+share files through the `/home` network directory.
+
+As a side note, since `/home` is a shared location for both build and runtime, and is also a 
+persistent storage location, some web apps use it to store state. We don't recomment this since
+it's easy to run into concurrent access issues. We recommend the use of services specifically
+designed for [storage][], which might also include backups, replication, and much more.
+
+[Kudu]: https://github.com/azure-App-Service/kudulite
+[storage]: https://azure.microsoft.com/en-us/product-categories/storage/
+
+## Node.js
+
+In general, node.js applications have a large number of package dependencies, either directly or indirectly,
+i.e. the dependencies of their dependencies. Since each package might contain several `.js` files, fetching 
```

Comment:
using `i.e.` is discouraged in the MS style guide: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/ie